packages/cardpay-sdk/sdk/safe-module.ts

```diff
@@ -34,48 +34,81 @@ import BN from 'bn.js';
 import JsonRpcProvider from '../providers/json-rpc-provider';
 import { camelCase } from 'lodash';
 
+/**
```

Comment:
This is a file that is a copy of a lot of scheduler work. I place it as @hidden because it is supposed to be common code shared between the scheduler and claimSettlement module but has not been refactored into the scheduler